XenModel/Actions/SupplementalPack/UploadSupplementalPackAction.cs

```diff
@@ -52,6 +52,8 @@ public class UploadSupplementalPackAction : AsyncAction
         private readonly List<Host> servers;
 
         private Pool_update poolUpdate = null;
+
+        public Dictionary<Pool_update, Dictionary<Host, SR>> SrUploadedUpdates = new Dictionary<Pool_update, Dictionary<Host, SR>>();

```

Comment:
A Dictionary<Host, SR> is sufficient here, as the update is already published as `PoolUpdate`